.golangci.yml

```diff
@@ -83,6 +83,7 @@ issues:
         - errcheck
         - dupl
         - gosec
+        - typecheck
```

Comment:
Based on golangci version update, you will want to remove "deadcode" and "varcheck" from linters and replace with "unused"

Reply:
Done